common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -29,11 +29,13 @@
     </li>
 }
 
-<div class="main-menu-container" id="main-menu">
+<label for="main-menu-toggle" class="main-menu-container__overlay" aria-hidden="true"></label>
```

Comment:
Since this test is for mobile only, I think tabbability or non-tabbability is out of scope, at least for this PR. What do you think?